frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/basicInfo.jspf

```diff
@@ -97,6 +97,19 @@
                       placeholder="Enter Description"><sw360:out value="${project.description}" stripNewlines="false"/></textarea>
         </td>
     </tr>
+    <tr>
+        <td width="33%">
+            <label class="checkboxlabel" for="enable_svm">
+                <input id="enable_svm" type="checkbox"
+                       name="<portlet:namespace/><%=Project._Fields.ENABLE_SVM%>"
+                       value="<sw360:out value="${project.enableSvm}"/>"
+                        <core_rt:if test="${project.enableSvm == 'true'}"> checked="checked" </core_rt:if>
```

Comment:
You could skip the `== 'true'` part in my opinion.

Reply:
You're right. That's a waste of some characters. Now corrected